### PR TITLE
Who we are

### PR DESCRIPTION
**Pull request by Lmise**
# What changes have you made
  - Added Content to the who we are section.
  - Added a view team button
  -Added padding to the content.

# Screenshots of the changes
## Desktop view
 
 
who-we-are1

## Mobile View
 
newmob

## GIFs
If some functionality changed or is introduced, attach a gif showing the changes or new feature

# Checks
  - [x] my code is well formated
  - [x] I only checked in the required files

# Request for a review
Remember to request for a review from team member

### CODE REVIEW

File: src/assets/scss/_index.scss [hunk 1]
```diff
@@ -6,7 +6,10 @@
 }
 #who-we-are{
   text-align: center;
-  padding:  2rem ;
+  padding: 2rem 6rem ;
+  a{
```
Comment (sigu) [inline src/assets/scss/_index.scss]: Have a space between the selector and the `{`

**Comment by GettyOrawo** [comment]
The padding for the mobile view is a bit off, maybe reduce the padding on both sides.

**Comment by GettyOrawo** [comment]
I prefer you using GIFs for a longer page

File: src/assets/scss/_index.scss (hunk 1 above)
Comment (Lmise) [reply src/assets/scss/_index.scss]: Okay. noted

File: src/assets/scss/_settings.scss [hunk 2]
```diff
@@ -100,6 +100,7 @@ $global-color-pick-contrast-tolerance: 0;
 $print-transparent-backgrounds: true;
 $canvas-background: #1a191e;
 $podii-orange: #ff5e0e; 
+$podii-blue:#008575;
```
Comment (sigu) [inline src/assets/scss/_settings.scss]: Put space between variable and value

**Pull request merged**